config/longrun_configs/longrun_moist_baroclinic_wave.yml

```diff
@@ -10,4 +10,4 @@ dt_save_state_to_disk: "10days"
 toml: [toml/longrun_baroclinic_wave.toml]
 diagnostics:
   - short_name: [pfull, wa, va, ua, ta, rhoa, rv, hus, ke, clw, cli]
-    period: 1days
+    period: 10days
```

Comment:
is this change intended?

Reply:
I thought it would be good to output that less frequently, given that the simulation is intended to run for 120 days in the long run test